install_dependencies.sh

```diff
@@ -29,10 +29,24 @@ if [ -f /etc/lsb-release ]; then
         x86_64_specific_packages=""
     fi
 
+    [[ "$( cat /etc/os-release | grep VERSION_ID= )" =~ ^VERSION_ID=\"(.*)\"$ ]]
+    DISTRO_VERSION=${BASH_REMATCH[1]}
+
+    # Ubuntu 18.04 provides only cmake 3.10 but >= 3.11 is required
+    if [ $DISTRO_VERSION = 18.04 ]; then
+        wget https://cmake.org/files/v3.12/cmake-3.12.3.tar.gz
+        tar xf cmake-3.12.3.tar.gz
+        cd cmake-3.12.3
+        ./configure
+        make -j16
+        sudo -E make install
```

Comment:
Are you sure that you don't break user system (and cmake) by doing this harmful operation?
Also there is no process / documentation how to recover after that.

What if user have already installed the latest CMake?

CMake must have enabled HTTPS support (or you will get "non supported protocol" during download of external files by OpenVINO scripts during build from sources at least).
How are you ensure that it is really enabled?

What kind of features really requires CMake 3.11 to build samples?

Reply:
You have a few good points. However, I just copied the block from the CentOS part (below in the same file) and thought this has been discussed before as CentOS should have the same issues you described.

> What kind of features really requires CMake 3.11 to build samples?

I have to take a look, I can't remember anymore.